### PR TITLE
Update reason for disabling Predictive Test Selection

### PR DESCRIPTION
**Pull request by marcphilipp**
Starting with v3.11.2 of the Gradle Enterprise Gradle plugin, PTS
detects unsupported test engines and fails test discovery for Kotest
since it's not yet supported.

### CODE REVIEW

File: test-kotest5/build.gradle [hunk 1]
```diff
@@ -41,11 +41,11 @@ dependencies {
 }
 
 test {
-    useJUnitPlatform()
-}
-
-// PTS just skips Kotest tests silently...
-tasks.withType(Test).configureEach {
+    useJUnitPlatform {
+        includeEngines("kotest")
```
Comment (melix) [inline test-kotest5/build.gradle:45]: Why is this needed?

Comment (marcphilipp) [reply test-kotest5/build.gradle:45]: It's not strictly necessary, but makes it explicit that only Kotest should be used by JUnit Platform. I noticed there are also [Spock](https://ge.micronaut.io/s/6fqdgpmvcgajq/dependencies?configurationFilter=WyJ0ZXN0UnVudGltZUNsYXNzcGF0aCJd&dependencies=spock&expandAll&focusedDependencyView=versions&toggled=W1swXSxbMl0sWzNdXQ) and the [JUnit Jupiter engine](https://ge.micronaut.io/s/6fqdgpmvcgajq/dependencies?configurationFilter=WyJ0ZXN0UnVudGltZUNsYXNzcGF0aCJd&dependencies=jupiter-engine&expandAll&focusedDependencyView=versions&toggled=W1swXSxbMl1d) on the test runtime classpath (see linked Build Scan).

File: test-kotest5/build.gradle [hunk 2]
```diff
@@ -41,11 +41,11 @@ dependencies {
 }
 
 test {
```
[the remark this answers is not in the document]
Comment (marcphilipp) [reply test-kotest5/build.gradle:43]: Would you like me to change that as part of this PR?

**Review by melix — APPROVED** [review]

**Comment by marcphilipp** [comment]
The failing check looks unrelated to my changes. Could someone please take a look? I suspect it's because this PR is from a fork and doesn't get secrets injected.

**Comment by melix** [comment]
Yes, that's correct @marcphilipp , don't worry about this :+1:
